client/app/scripts/liveblog-themes/module.js

```diff
@@ -131,12 +131,7 @@ import listTpl from 'scripts/liveblog-themes/views/list.ng1';
                 }
 
                 themes.forEach((theme) => {
-                    // create criteria to load blogs with the theme.
-                    const criteria = {
-                        where: JSON.stringify({'blog_preferences.theme': theme.name}),
-                    };
-
-                    api.blogs.query(criteria).then((data) => {
+                    api.get(`/count_theme_blogs/${theme.name}`).then((data) => {
```

Comment:
We can avoid the extra requests/hits by attaching this info in the "on_fetched" method from the blog service. Like,  

```python
theme["blog_count"] = blog_service.get_from_mongo(...
```

Reply:
Oh yeah, that makes more sense.